cvat/apps/lambda_manager/views.py

```diff
@@ -0,0 +1,307 @@
+from django.shortcuts import render
+from rest_framework import viewsets, status
+from rest_framework.response import Response
+from cvat.apps.engine.frame_provider import FrameProvider
+from cvat.apps.engine.models import Task as TaskModel
+from cvat.apps.dataset_manager.task import put_task_data, patch_task_data
+from django.core.exceptions import ValidationError
+import base64
+import json
+import requests
+import django_rq
+import rq
+from functools import wraps
+from cvat.apps.engine.serializers import LabeledDataSerializer
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+
+class LambdaGateway:
+    NUCLIO_ROOT_URL = '/api/functions'
+
+    def _http(self, method="get", scheme=None, host=None, port=None,
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5729179)